Docs/source/usage/parameters.rst

```diff
@@ -1764,6 +1764,11 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Whether to include the guard cells in the output of the raw fields.
     Only works with ``<diag_name>.format = plotfile``.
 
+    * ``<diag_name>.plot_raw_rho`` (`0` or `1`) optional (default `0`)
+    By default, the charge density written in the plot files is averaged on the cell centers.
+    When ``<diag_name>.plot_raw_rho = 1``, then the raw (i.e. non-averaged) charge density also saved in the output files.
```

Comment:
```suggestion
    When ``<diag_name>.plot_raw_rho = 1``, then the raw (i.e. non-averaged) charge density is also saved in the output files.
```

Reply:
Does this mean that both the averaged and the non-averaged fields are saved on disk?

Reply:
Yes, provided that `rho` is included in `fields_to_plot` (both averaged and non-averaged are written independently).
Btw, I have tested this option and it seems to work.